Tools/_sunrise/update_pr_branches.ps1

```
@@ -0,0 +1,251 @@
+        New-Item -ItemType Directory -Path $parent | Out-Null
+    }
+
+    return $RequestedPath
+}
+
+function Write-Log {
+    param([string] $Message = "")
+
+    Write-Host $Message
+    Add-Content -LiteralPath $script:ResolvedLogPath -Value $Message -Encoding UTF8
+}
```

PSScriptAnalyzer (1.25.0)

[warning] 'Write-Log' is a cmdlet that is included with PowerShell (version core-6.1.0-windows) whose definition should not be overridden

(PSAvoidOverwritingBuiltInCmdlets)